src/lib/services/plausible.ts

```diff
@@ -0,0 +1,50 @@
+import axios from "axios";
+import fs from "fs";
+import path from "path";
+import os from "os";
+
+export interface PlausibleEvent {
+  name: string;
+  properties: Record<string, string | number | boolean>;
+}
+
+const CONFIG_FILE_PATH = path.resolve(os.homedir(), ".genlayer-config.json");
+
+export class PlausibleService {
+  private apiUrl: string;
+  private siteId: string;
+
+  constructor(siteId = 'studio.genlayer.com', apiUrl = "https://plausible.io/api/event") {
```

Comment:
@epsjunior this should be posted to something like cli.genlayer.com so the events doesn't get fixed with the Studio events